inst/@sym/private/const_to_python_str.m

```diff
@@ -0,0 +1,99 @@
+%% Copyright (C) 2015, 2016 Colin B. Macdonald
+%% Copyright (C) 2016 Lagu
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @deftypefun {[@var{s}, @var{flag}] =} const_to_python_str (@var{x})
+%% Recognize constant and return their python string equivalent.
+%%
+%% Private helper function.
+%%
+%% Caution: there are two copies of this file for technical
+%% reasons: make sure you modify both of them!
+%%
+%% @seealso{sym, vpa}
+%% @end deftypefun
+
+function [s, flag] = const_to_python_str (x)
+
+  persistent list
+  persistent const
+
+  if (isempty (list))
+    % Table of special doubles and special strings
+    % format: number, {octave strings}, resulting python expression
```

Comment:
Hmm maybe I did by accident.  Not sure what comment you mean?  Was it in your #594 or in master?